internal/services/applicationinsights/application_insights_standard_webtests_resource_test.go

```diff
@@ -59,6 +68,45 @@ func (ApplicationInsightsStandardWebTestResource) Exists(ctx context.Context, cl
 	return utils.Bool(resp.Model != nil && resp.Model.Properties != nil), nil
 }
 
+func (ApplicationInsightsStandardWebTestResource) sslCheckConfig(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-appinsights-%d"
+  location = "%s"
+}
+
+resource "azurerm_application_insights" "test" {
+  name                = "acctestappinsights-%d"
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+  application_type    = "web"
+}
+
+resource "azurerm_application_insights_standard_web_test" "test" {
+  name                    = "acctestappinsightswebtests-%d"
+  location                = azurerm_resource_group.test.location
+  resource_group_name     = azurerm_resource_group.test.name
+  application_insights_id = azurerm_application_insights.test.id
+  geo_locations           = ["us-tx-sn1-azr"]
+
+  request {
+    url = "https://microsoft.com"
+  }
+	validation_rules {
```

Comment:
The formatting is off here.

Reply:
Ouch, fixed it. Is there a reason formatting is not checked anymore by the GitHub Actions?

Reply:
I believe we turned them off as there was a bug in that linter. We'll get it back on soon!